### PR TITLE
Feature/authentication20

### PR DESCRIPTION
**Pull request by Jan-Bart**

### CODE REVIEW

File: test/logoutCallback.js [hunk 1]
```diff
@@ -48,43 +48,6 @@ describe('GET /logout/:serviceProvider/callback', function onDescribe() {
...
-
-    res.on('end', () => {
-      assert(res.sendStatus.calledWith(401));
-      return done();
-    });
-
-    router.handle(req, res);
-  });
-
   it('should 401 when serviceprovider unknown', function onIt(done) {
```
Comment (vademo) [inline test/logoutCallback.js]: Your test name doesn't align with your test

File: lib/controller.js [hunk 2]
```diff
@@ -180,26 +188,21 @@ module.exports = function createController(config) {
 
   function logoutCallback(req, res) {
     const serviceName = req.params.service;
-    const state = req.query.state;
-    const logoutState = req.session[`${serviceName}_logoutKey`];
-
-    if (state !== logoutState) {
```
Comment (vademo) [inline lib/controller.js]: Has this become irrelevant?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digipolis/auth",
-  "version": "1.4.1",
+  "version": "1.4.3",
```
Comment (vademo) [inline package.json]: 1.4.2 seems to be missing ?
Also I would do a minor or even major because there are some behavioural changes.

**Comment by vademo** [comment]
Burning the midnight oil? 💪

File: package.json (hunk 3 above)
Comment (Jan-Bart) [reply package.json]: 1.4.2 does exists here: https://github.com/digipolisantwerp/auth_module_nodejs/tree/v1.4.2

Fair point. It should be backwards compatible and (should have) none breaking changes.
So a minor would ok?

File: test/logoutCallback.js (hunk 1 above)
Comment (Jan-Bart) [reply test/logoutCallback.js]: Fixed. Will push it after the other remarks are resolved.

File: package.json (hunk 3 above)
Comment (vademo) [reply package.json]: If it's backwards compatible a minor is correct.

Comment (Jan-Bart) [reply package.json]: Fixed (and pushed)

File: lib/controller.js (hunk 2 above)
Comment (Jan-Bart) [reply lib/controller.js]: For now it is. It can be added later if we need it.

**Pull request merged**